server/src/main/resources/db/migration/V1_269__add_partner_source_partner_and_job_creator_fields.sql

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2022 Talent Beyond Boundaries.
+ * Copyright (c) 2023 Talent Beyond Boundaries.
```

Comment:
Cool - we should really do this throughout. I've opened a ticket for it.

Reply:
Intellij has support for autogenerating and updating this. I didn't put that change in - Intellij did.